data/json/monsters/feral_humans.json

```diff
@@ -59,6 +59,26 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_chud",
+    "type": "MONSTER",
+    "name": { "str": "feral troglobite" },
+    "description": "One of the still-living crazed humans, changed by something they are exposed to down here.  Their face is dominated by unnaturally large, faintly glowing eyes and needle-sharp teeth poking through their lips.  They remind you of something out of a campy monster movie - and of the good times when underground cannibals were less of a real concern.",
+    "copy-from": "mon_feral_human_pipe",
+    "looks_like": "mon_zombie_runner",
+    "symbol": "@",
+    "color": "white",
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 3,
+    "dodge": 4,
+    "vision_night": 15,
+    "special_attacks": [ { "id": "scratch", "min_mul": 0.5, "cooldown": 10 }, { "type": "bite", "min_mul": 0.7, "cooldown": 15 } ],
+    "death_drops": "default_zombie_death_drops",
+    "zombify_into": "mon_zombie_shady",
```

Comment:
Ah, good, I've been thinking about advanced zombies from ferals. (Would it be possible to use a monster group, with time settings?)

Reply:
Yes, I considered adding an evolution to the generic zombie group for all ferals to show them amortizing but it felt a bit too tangential to underground livin'.